src/components/Blog/AutoCompletedSearhBlog.tsx

```diff
@@ -32,37 +40,53 @@ const AutoCompletedSearhBlog = () => {
       }
     };
     fetchAutoBlogList();
+
+    if (isInitialLoad) {
+      const tagLabel = blogAutoLists.find((item) => item.id === Number(tagID));
```

Comment:
So need tagID to check if they are in blogAutoLists, we will display UI of blog lists in the page @JohnMulligan